docs/deployment.md

```diff
@@ -38,17 +38,17 @@ pnpm localenv:stop && pnpm localenv:dbvolumes:remove
 pnpm localenv:start
 ```
 
-### Exploring Accounts on Mock account provider
+### Exploring Accounts on Mock Account Servicing Entity
 
-Navigate to `localhost:3030` to view the accounts on one instance of the Mock Account Provider called Cloud Nine Wallet.
+Navigate to `localhost:3030` to view the accounts on one instance of the Mock Account Servicing Entity called Cloud Nine Wallet.
 
-![Mock Account Provider Accounts](./img/map-accounts.png)
+![Mock Account Servicing Entity Accounts](./img/map-accounts.png)
 
 The accounts of the second instance (Happy Life Bank) can be found on `localhost:3031`.
 
 When clicking on the Account Name, a list of Transactions appears.
 
-![Mock Account Provider Transactions](./img/map-transactions.png)
+![Mock Account Servicing Entity Transactions](./img/map-transactions.png)
 
 ### Admin APIs
 
```

Comment:
In the Open Payments APIs section, can you please update the eCommerce example section to include the changes I made for splitting out the quote grant creation 🙃